infra/modules/worklytics-connectors-msft-365/main.tf

```diff
@@ -1,7 +1,3 @@
-# NOTE: region used to be passed in as a variable; put it MUST match the region in which the lambda
-# is provisioned, and that's implicit in the provider - so we should just infer from the provider
-data "aws_region" "current" {}
```

Comment:
BTW I've forgot to comment this; not sure why it was here but I've removed it. I think is not going to break AWS implementation as this variable is not used here, but just for double checking @eschultink

Reply:
maybe to pull aws account ID or ARN of the user who's running terraform, or something like that. If stuff still works, shouldn't be a problem.

likely a relic of when we did the cognito stuff in there.